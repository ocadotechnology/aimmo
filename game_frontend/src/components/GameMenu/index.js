import styled from 'styled-components'

const GameMenu = styled.nav`
  background-color: ${props => props.theme.palette.background.default}
  grid-area: game-menu
`

export default GameMenu
