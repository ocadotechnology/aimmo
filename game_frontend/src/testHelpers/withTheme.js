import React from 'react'
import { createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles'
import { ThemeProvider } from 'styled-components'
import theme from '../theme'

const muiTheme = createMuiTheme(theme)

const withTheme = component => {
  return (
    <ThemeProvider theme={muiTheme}>
      <MuiThemeProvider theme={muiTheme}>
        {component}
      </MuiThemeProvider>
    </ThemeProvider>
  )
}

export default withTheme
