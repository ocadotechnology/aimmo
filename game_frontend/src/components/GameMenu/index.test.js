/* eslint-env jest */
import React from 'react'
import GameMenu from 'components/GameMenu'
import renderer from 'react-test-renderer'
import withTheme from 'testHelpers/withTheme'

describe('<GameMenu />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(withTheme(<GameMenu />)).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
