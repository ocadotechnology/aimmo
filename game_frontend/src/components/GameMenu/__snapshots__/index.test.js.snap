// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameMenu /> renders correctly 1`] = `
.c0 {
  background-color: #303030;
  grid-area: game-menu;
}

<nav
  className="c0"
/>
`;
